src/components/courses/HandInExamsPage.tsx

```
@@ -0,0 +1,72 @@
+    if (userJson) user = JSON.parse(userJson);
+    formData.append("userId", user.id);
+    formData.append("examId", state.exam.id);
+    submissionApi.submit(formData);
+  };
+
+  return (
+    <div className="container mx-auto mt-8 p-4">
+      <div className="w-full max-w-2xl bg-slate-300 dark:bg-slate-800 shadow-md rounded-lg p-6 mx-auto">
+        <div className="flex justify-between items-center mb-4">
+          <Link to={`/course/${state.courseId}`} state={state.courseId}>
+            <button className="flex items-center text-blue-500 hover:text-blue-600 transition duration-300">
```

Biome

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)